src/workloads/execution/MixedMultiDeletesBatched.yml

```diff
@@ -0,0 +1,37 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Deletes a range of documents using the BATCHED_DELETE query exec stage both in isolation and while performing writes on another collection. Tests deletes on documents of size ~1KB then deletes on documents of ~10MB.
+
+Keywords:
+- RunCommand
+- Loader
+- LoggingActor
+- CrudActor
+- insert
+- delete
+- batch
+- deleteMany
+- latency
+
+Clients:
+  Default:
+    QueryOptions:
+      # Allow for longer duration since the loading of the data may take a while.
+      socketTimeoutMS: 3_600_000  # = 1 hour
+
+LoadConfig:
```

Comment:
Would it be possible to parametrise the `MixedMultiDeletes` file, and do a `LoadConfig` for each document size?
I don't remember exactly, but I think it might be possible.

https://github.com/mongodb/genny/blob/91716090002bc1a76155c350bdf9fd17531f97d9/src/lamplib/src/genny/tasks/preprocess.py#L235